data/geo_groups.json

```diff
@@ -0,0 +1,12 @@
+{
+  "CO": {
+    "co-platte-river-power-authority": {
+      "utilities": [
+        "co-estes-park-power-and-communications",
+        "co-fort-collins-utilities",
+        "co-longmont-power-and-communications",
+        "co-loveland-water-and-power"
+      ]
```

Comment:
A pre-requisite for adding utilities/cities/counties here would be that they're already defined as authorities on their own, right?